### PR TITLE
Update audio-conferencing-with-direct-routing-for-gcch-and-dod.md

### PR DESCRIPTION
**Pull request by ruizoscar**
Including a configuration step that is missing from the documentation

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md [hunk 1]
```diff
@@ -86,23 +86,77 @@ You can see the ID of your Audio Conferencing Bridge using Get-CsOnlineDialInCon
...
+
+> [!NOTE]
+> Location-Based Routing isn't available in Microsoft 365 Government Community Cloud (GCC) High or DoD deployments. When enabling Audio Conferencing, please verify that no Audio Conferencing users in the GCC High or the DoD environments are enabled for Location-Based Routing.
+
+#### Defining a global voice routing policy
+
+A global voice routing policy can be defined by defining a PSTN usage, a voice route, a voice routing policy, and assigning the new voice routing policy as the global voice routing policy of your organization.
+
+The following steps describe how to define a new global voice routing policy for an organization without one, if your organization already has voice routing policies defined, please verify that the following configuration doesn’t conflict with the existing voice routing policies of your organization.
```
Comment (JohanFreelancer9) [inline Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md:101]:
```suggestion
The following steps describe how to define a new global voice routing policy for an organization without one. If your organization already has voice routing policies defined, please verify that the following configuration doesn’t conflict with the existing voice routing policies of your organization.
```

File: Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md [hunk 2]
```diff
@@ -86,23 +86,77 @@ You can see the ID of your Audio Conferencing Bridge using Get-CsOnlineDialInCon
...
+
+  ```PowerShell
+  New-CsOnlineVoiceRoute -Identity "International" -NumberPattern ".*" -OnlinePstnGatewayList sbc1.contoso.biz -OnlinePstnUsages "International"
+  ```
+
+When defining a new voice route for your organization, please specify one or multiple of the PSTN online PSTN gateways that have been defined for your organization during the configuration of Direct Routing. 
+
+The number pattern specifies which calls will be routed through the specified list of gateways based on the destination phone number of the call. In the example above, calls to any destinations in the world will match the voice route. If you would like to restrict the phone numbers that can be dialed from the meetings of users in your organization, you can change the number pattern to have the voice route match only the number patterns of the destinations allowed. Please note that if there are no voice routes that match the number pattern of the destination phone number of a given call, the call will not be routed.
+
+For additional information, see [New-CsOnlineVoiceRoute](https://docs.microsoft.com/powershell/module/skype/new-csonlinevoiceroute)
```
Comment (JohanFreelancer9) [inline Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md:121]:
```suggestion
For additional information, see [New-CsOnlineVoiceRoute](https://docs.microsoft.com/powershell/module/skype/new-csonlinevoiceroute).
```

File: Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md [hunk 3]
```diff
@@ -86,23 +86,77 @@ You can see the ID of your Audio Conferencing Bridge using Get-CsOnlineDialInCon
...
+
+To create a new voice routing policy, enter:
+
+  ```PowerShell
+  New-CsOnlineVoiceRoutingPolicy "InternationalVoiceRoutingPolicy" -OnlinePstnUsages "International"
+  ```
+
+If multiple PSTN usages are being defined in the voice routing policy, they will be evaluated in the order in which they are defined. It is recommended that the PSTN usages are defined in the order of the most specific to the more generic in terms of the number patterns of the voice routes associated with the PSTN usages. For example, if a PSTN usage was defined to route calls to the United States, and another PSTN usage was defined to route calls to any other location in the world, the PSTN usage for calls to the United States should be listed in the voice routing policy ahead of the PSTN usage to route calls to any other location in the world.
+
+For additional information, see [New-CsOnlineVoiceRoutingPolicy](https://docs.microsoft.com/powershell/module/skype/new-csonlinevoiceroutingpolicy)
```
Comment (JohanFreelancer9) [inline Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md:131]:
```suggestion
For additional information, see [New-CsOnlineVoiceRoutingPolicy](https://docs.microsoft.com/powershell/module/skype/new-csonlinevoiceroutingpolicy).
```

File: Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md [hunk 4]
```diff
@@ -86,23 +86,77 @@ You can see the ID of your Audio Conferencing Bridge using Get-CsOnlineDialInCon
...
+
+To assign the new voice route to the global voice routing policy of your organization, enter:
+
+  ```PowerShell
+  Grant-CsOnlineVoiceRoutingPolicy -PolicyName "InternationalVoiceRoutingPolicy" -Global
+  ```
+
+For additional information, see [Grant-CsOnlineVoiceRoutingPolicy](https://docs.microsoft.com/powershell/module/skype/grant-csonlinevoiceroutingpolicy)
```
Comment (JohanFreelancer9) [inline Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md:139]:
```suggestion
For additional information, see [Grant-CsOnlineVoiceRoutingPolicy](https://docs.microsoft.com/powershell/module/skype/grant-csonlinevoiceroutingpolicy).
```

File: Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md [hunk 5]
```diff
@@ -86,23 +86,77 @@ You can see the ID of your Audio Conferencing Bridge using Get-CsOnlineDialInCon
...
+Once the global voice routing policy has been defined, any outbound calls made from meetings organized by users in your organization will be evaluated against the voice routes associated to the PSTN usages of the global voice routing policy. The outbound calls will be routed according to the first voice route that matches the number pattern of the dialed phone number.
+
+### Step 5: Assign Audio Conferencing with Direct Routing for GCC High or DoD licenses to your users
 
 To assign Audio Conferencing with Direct Routing for GCC High or DoD licenses to your user, see [Assign licenses to users](https://docs.microsoft.com/microsoft-365/admin/manage/assign-licenses-to-users).
 
-### Step 5: (Optional) See a list of Audio Conferencing numbers in Teams
+### Step 6: (Optional) See a list of Audio Conferencing numbers in Teams
 
 To see the list of Audio Conferencing numbers of your organization, go to [See a list of Audio Conferencing numbers in Microsoft Teams](see-a-list-of-audio-conferencing-numbers-in-teams.md)
```
Comment (JohanFreelancer9) [inline Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md:149]:
```suggestion
To see the list of Audio Conferencing numbers of your organization, go to [See a list of Audio Conferencing numbers in Microsoft Teams](see-a-list-of-audio-conferencing-numbers-in-teams.md).
```

File: Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md [hunk 6]
```diff
@@ -86,23 +86,77 @@ You can see the ID of your Audio Conferencing Bridge using Get-CsOnlineDialInCon
...
 
-### Step 6: (Optional) Set auto attendant languages for the Audio Conferencing dial-in numbers of you organization
+### Step 7: (Optional) Set auto attendant languages for the Audio Conferencing dial-in numbers of you organization
 
 To change the languages of the Audio Conferencing dial-in numbers of your organization, see [Set auto attendant languages for Audio Conferencing in Microsoft Teams](set-auto-attendant-languages-for-audio-conferencing-in-teams.md)
```
Comment (JohanFreelancer9) [inline Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md:153]:
```suggestion
To change the languages of the Audio Conferencing dial-in numbers of your organization, see [Set auto attendant languages for Audio Conferencing in Microsoft Teams](set-auto-attendant-languages-for-audio-conferencing-in-teams.md).
```

File: Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md [hunk 7]
```diff
@@ -86,23 +86,77 @@ You can see the ID of your Audio Conferencing Bridge using Get-CsOnlineDialInCon
...
 
-### Step 7: (Optional) Change the settings of the Audio Conferencing bridge of your organization
+### Step 8: (Optional) Change the settings of the Audio Conferencing bridge of your organization
 
 To change the settings of the Audio Conferencing bridge of your organization, see [Change the settings for an Audio Conferencing bridge](change-the-settings-for-an-audio-conferencing-bridge.md)
```
Comment (JohanFreelancer9) [inline Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md:157]:
```suggestion
To change the settings of the Audio Conferencing bridge of your organization, see [Change the settings for an Audio Conferencing bridge](change-the-settings-for-an-audio-conferencing-bridge.md).
```

File: Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md [hunk 8]
```diff
@@ -86,23 +86,77 @@ You can see the ID of your Audio Conferencing Bridge using Get-CsOnlineDialInCon
...
 
-### Step 8: (Optional) Set the phone numbers included in the meeting invites of the users in your organization
+### Step 9: (Optional) Set the phone numbers included in the meeting invites of the users in your organization
 
 To change the set of phone numbers that are included in the meeting invites of the users is your organization, see [Set the phone numbers included on invites in Microsoft Teams](set-the-phone-numbers-included-on-invites-in-teams.md)
```
Comment (JohanFreelancer9) [inline Teams/audio-conferencing-with-direct-routing-for-gcch-and-dod.md:161]:
```suggestion
To change the set of phone numbers that are included in the meeting invites of the users is your organization, see [Set the phone numbers included on invites in Microsoft Teams](set-the-phone-numbers-included-on-invites-in-teams.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**